.ci/azure-pipelines-daily.yml

```diff
@@ -0,0 +1,41 @@
+schedules:
+  - cron: '0 10 * * *'
+    displayName: Daily late night PST (GMT-8)
+    branches:
+      include:
+        - helics3
+
+jobs:
+  - job: Daily Linux
+    strategy:
+      matrix:
+        all-tests:
+          containerImage: 'helics/buildenv:ubuntu18.04-default-builder'
+          test_config: 'daily'
+        zmq-subproject:
+          containerImage: 'helics/buildenv:ubuntu18.04-default-builder'
+          test_config: 'ci'
+          zmq_subproject: true
+    pool:
+      vmImage: 'ubuntu-latest'
+    container: $[ variables['containerImage'] ]
+    timeoutInMinutes: 90
+
+    steps:
+      - bash: |
+          source scripts/setup-helics-ci-options.sh
+          mkdir -p build && cd build
+          ../scripts/ci-build.sh
+        env:
+          MAKEFLAGS: '-j 4'
+          CMAKE_GENERATOR: 'Unix Makefiles'
```

Comment:
should we have a test somewhere with the ninja generator?  Not that this needs to be.

Reply:
The builds on Drone CI use Ninja, but being able to have some of the tests use different generators in the future was one of the reasons for selecting the generator using an environment variable.